src/main/resources/mixins.notenoughupdates.json

```diff
@@ -37,8 +38,8 @@
     "MixinMinecraft",
     "MixinMouseHelper",
     "MixinNetHandlerPlayClient",
-    "MixinOptifineCustomItems",
     "MixinOFDynamicLights",
+    "MixinOptifineCustomItems",
```

Comment:
?

Reply:
i didnt touch that file, intellij moment